src/ext/java/org/opentripplanner/ext/actuator/ActuatorAPI.java

```diff
@@ -130,9 +127,9 @@ public Response actuator(@Context UriInfo uriInfo) {
     @Path("/health")
     public Response health() {
         if (router.graph.updaterManager != null) {
-            Collection<String> waitingUpdaters = router.graph.updaterManager.waitingUpdaters();
+            int waitingUpdaters = router.graph.updaterManager.numberOfNonePrimedUpdaters();
 
-            if (!waitingUpdaters.isEmpty()) {
+            if (waitingUpdaters > 0) {
                 LOG.info("Graph ready, waiting for updaters: {}", waitingUpdaters);
```

Comment:
By changing `waitingUpdaters` to an int, we lose the information on _which_ updaters that are still pending. This is useful information when debugging e.g. startup-times.

Example from todays log:
`Graph ready, waiting for updaters: [siri-et-google-pubsub-updater] `

Reply:
Good point, I will fix that.